tests/typescript_comments/methods.ts

```diff
@@ -0,0 +1,38 @@
+export class Point {
+/**
+ * Does something.
+ */
+  foo() {}
+
+    /**
+     * Does something else.
+     */
+  bar() {}
+
+                /**
+                 * Does
+                 * something
+                 * much
+                 * better
+                 * than
+                 * the
+                 * rest.
+                 */
+  baz() {}
+
+      /**
+       * Buzz-Fizz.
+       * Note: This is indented too far.
+       */
+      fizzBuzz() {}
+
+      /**
+       * Turns the given string into pig-latin.
+       */
+              pigLatinize(value: string) {
+/**
+ * This is a block comment inside of a method.
+ */
```

Comment:
Can we add a test where the stars aren’t lined up in the input?

```js
/**
* first line
 * second line
* third
    * fourth
  */
function foo() {}
```

Reply:
@lydell Sure thing!